app/data/views.py

```diff
@@ -29,12 +32,49 @@ def list(self, request, *args, **kwargs):
         return response
 
     @list_route(methods=['get'])
+    def stepwise(self, request):
+        """Return an aggregation counts the occurrence of events per week (per year) between
+        two bounding datetimes
+        """
+        # We'll need to have minimum and maximum dates specified to properly construct our SQL
+        try:
+            start_date = parse_date(request.GET['occurred_min'])
```

Comment:
I think `request.query_params` is preferred over `request.GET`, at least in DRF-world.